modules/apps/adaptive-media/adaptive-media-web/build.gradle

```diff
@@ -1,8 +1,4 @@
-buildCSS {
-	imports = [
-		new File(npmInstall.nodeModulesDir, "@clayui/css/src/scss")
```

Comment:
This means that `@clayui/css` is added automatically to the list of Sass dependencies and there's no need to add it manually anymore?

Reply:
@jbalsas correct, we have this list of sass dependencies that are automatically added [here](https://github.com/liferay/liferay-frontend-projects/blob/master/projects/npm-tools/packages/npm-scripts/src/presets/standard/index.js#L56-L60).

Ideally we remove bourbon once we get modules to stop referencing it.